pkg/services/correlations/api.go

```diff
@@ -298,8 +298,23 @@ type GetCorrelationsBySourceUIDResponse struct {
 // 404: notFoundError
 // 500: internalServerError
 func (s *CorrelationsService) getCorrelationsHandler(c *contextmodel.ReqContext) response.Response {
+	limit := c.QueryInt64("limit")
+	if limit <= 0 {
+		limit = 100
+	}
+
+	page := c.QueryInt64("page")
+	if page <= 0 {
+		page = 1
+	}
+
+	sourceUIDs := c.QueryStrings("sourceuid")
```

Comment:
In Correlations API responses it's called `sourceUID` so maybe we could make it consistent[^1]? 

[^1]: It's not really standardized, though, and all over the place with patterns like `org_id`, `connectionUid` and`dashboardUID` 🤦

Reply:
in https://github.com/grafana/grafana/pull/51801#discussion_r930165948 i got asked to use `UID` as it was being standardized, however it is not really enforced anywhere else, but +1